Example/ViewController.swift

```diff
@@ -62,6 +62,21 @@ class ViewController: UIViewController {
     }
   }
 
+  func fetchDecodable() {
```

Comment:
If we're being consistent should this technically be `fetchDecodableExample()`? (based on other functions in this class)